examples/python/image_storage.py

```diff
@@ -0,0 +1,47 @@
+# This example shows
+# 1 - how to load the TensorFlow Flowers Images into a dataframe
+# 2 - manipulate the dataframe
+# 3 - write the dataframe to a Delta Lake table
+# 4 - read the new Delta Lake table
+
+import pyspark.sql.functions as fn
+import pyspark
+from delta import configure_spark_with_delta_pip
+
+builder = (
+  pyspark.sql.SparkSession.builder
+    .appName('quickstart')
+    .config('spark.sql.extensions', 'io.delta.sql.DeltaSparkSessionExtension')
+    .config('spark.sql.catalog.spark_catalog', 'org.apache.spark.sql.delta.catalog.DeltaCatalog')
+)
+
+spark = configure_spark_with_delta_pip(builder).getOrCreate()
```

Comment:
Since this example was requested, I change my mind, I actually like setting it up the way you did so it can be run directly. Can we change it back to using `configure_spark_with_delta_pip`?

But can we also provide something like "to run this example yourself, run `python3 ...`?

Reply:
If you want to run it as part of the integration tests the commands are
`python3  run-integration-tests.py --version 1.1.0 --python-only --test image_storage` to run just this or 
`python3  run-integration-tests.py --version 1.1.0 --python-only` to run all of them

I did run them successfully though :)

Reply:
@allisonport-db I changed it to include a comment on set up and commented out the test session. Would you like me to remove the test session altogether?